### ISSUE TITLE
add Deep Learning for Coders by Fast.AI

### ISSUE THREAD

**Issue by apolmig** [dair-ai/ML-YouTube-Courses#6]
**Fast.AI Practical Deep Learning for Coders**
To learn applied DL from 0 to hero. After finishing this course you will know:

- Computer vision, including image classification
- Natural language processing (NLP), including document classification
- Collaborative filtering
- How to turn your models into web applications
- Why and how deep learning models work
- How to implement stochastic gradient descent and a complete training loop from scratch
- How to think about the ethical implications of your work
- 
🔗 [Link to Course](https://www.youtube.com/playlist?list=PLfYUBJiXbdtRL3FMB3GoWHRI8ieU6FhfM)

**Comment by omarsar** [comment]
added! Thanks
